src/events.js

```diff
@@ -55,11 +56,17 @@ module.exports = function(ctx) {
     currentMode.mousedown(event);
   };
 
+  events.drawChanged = function(event) {
+    featuresChanged = event.features;
+  }
+
   events.mouseup = function(event) {
     mouseDownInfo.isDown = false;
     var target = getFeatureAtAndSetCursors(event, ctx);
     event.featureTarget = target;
 
+    ctx.map.fire('draw.modifiedFeature', {features: featuresChanged});
```

Comment:
@ghoshkaj when I was playing with this I found a bug where `draw.modfied` was firing (with features) even though `draw.changed` had not fired. We could also clear up `featuresChanged` just after firing. This might be the best idea.